src/pages/FlagLabware.tsx

```diff
@@ -0,0 +1,183 @@
+  }, []);
+
+  const [relatedFlags, setRelatedFlags] = React.useState<FlagDetail[]>([]);
+
+  const checkRelatedFlagges = useCallback(
+    async (labwares: LabwareFieldsFragment[], foundLabware: LabwareFieldsFragment): Promise<string[]> => {
+      return stanCore
+        .GetLabwareFlagDetails({
+          barcodes: [foundLabware.barcode]
+        })
+        .then((response) => {
+          setRelatedFlags(response.labwareFlagDetails);
+          return [];
+        });
+    },
+    [stanCore]
+  );
+
+  return (
+    <AppShell>
+      <AppShell.Header>
+        <AppShell.Title>Flag Labware</AppShell.Title>
+      </AppShell.Header>
+      <AppShell.Main>
+        {serverError && <Warning error={serverError} />}
+        <div className="max-w-screen-xl mx-auto">
+          <Formik initialValues={initialValues} validationSchema={validationSchema} onSubmit={convertValuesAndSubmit}>
+            {({ values, setFieldValue }) => (
+              <Form>
+                <GrayBox>
+                  <div className="md:w-3/4">
+                    <div className="space-y-4">
+                      {serverError && <Warning error={serverError} />}
+                      <Heading level={3}>Labware</Heading>
+                      <MutedText>Please scan in the labware you wish to flag.</MutedText>
+                      <LabwareScanner
+                        limit={1}
+                        onAdd={(lw) => {
+                          setFieldValue('labware', lw);
+                        }}
+                        labwareCheckFunction={checkRelatedFlagges}
+                      >
+                        {({ labwares, removeLabware }) =>
+                          labwares.map((labware, index) => (
+                            <Panel key={labware.barcode}>
+                              <div className="flex flex-row items-center justify-end">
+                                {
+                                  <RemoveButton
+                                    data-testid={'remove'}
+                                    onClick={() => {
+                                      setFieldValue('labware', undefined);
+                                      removeLabware(labware.barcode);
+                                    }}
+                                  />
+                                }
+                              </div>
+
+                              <DataTable columns={labwareColumns} data={[labware]} />
+
+                              {relatedFlags.length > 0 && <LabwareFlagDetails flagDetails={relatedFlags} />}
+
+                              <div className="mt-6">
+                                <Heading level={3}>Description</Heading>
+                                <MutedText>Please enter a reason for flagging the labware.</MutedText>
+
+                                <Field
+                                  as={'textarea'}
+                                  placeholder="Enter your the reason for flagging the labware here."
```

Comment:
Is it required to have both MutedText and placeholder with the same information?

Reply:
not required, I will remove the placeholder and keep the muted text, so it is coherent with the other pages